cli-tests/src/test.rs

```diff
@@ -252,14 +249,14 @@ async fn test_command_succeeds_with_bundle_using_bep() {
     .failure();
 
     let requests = state.requests.lock().unwrap().clone();
-    assert_eq!(requests.len(), 4);
-
-    let tar_extract_directory = assert_matches!(&requests[2], RequestPayload::S3Upload(d) => d);
+    assert_eq!(requests.len(), 3);
+    assert_matches!(requests[0], RequestPayload::CreateBundleUpload(_));
+    let tar_extract_directory = assert_matches!(&requests[1], RequestPayload::S3Upload(d) => d);
+    assert_matches!(requests[2], RequestPayload::TelemetryUploadMetrics(_));
 
     let junit_file = fs::File::open(tar_extract_directory.join("junit/0")).unwrap();
     let junit_reader = BufReader::new(junit_file);
 
-    // Uploaded file is a junit, even when using BEP
```

Comment:
Most comments in tests have been for self-explanatory code adding to the noise. Additionally, some of them have been incorrect, stale, or added non-relevant context.

Removing them and making the test assertions clearer is better.